alterconfigs.go

```diff
@@ -0,0 +1,202 @@
+package kafka
+
+import (
+	"bufio"
+	"time"
+)
+
+// See https://kafka.apache.org/protocol#The_Messages_AlterConfigs
+type alterConfigsRequestV0 struct {
```

Comment:
high level question: the protocol implementation has been vastly simplified in the kafka-go `0.4` branch, thoughts about changing this PR to branch off of `0.4` instead to take advantage of that?

Reply:
sorry, I don't know about this infor, i always think the master is latest version.

Reply:
Hey, I have updated this to 0.4 branch. here is PR [544](https://github.com/segmentio/kafka-go/pull/544). could you help review?